headlamp-k8s/headlamp
# headlamp-plugin-0.6.0, latest api docs, plugin/examples updated

- docs api: Generate latest API reference
- headlamp-plugin: Bump version v0.6.0
- plugins/examples: Upgrade headlamp-plugin to 0.6.0

# how to test

1. Try some checks and run a plugin:
```bash
cd plugins/examples/pod-counter
npm install
npm run tsc && npm run lint && CI=true npm run test
npm start
```
2. See the plugin inside headlamp
